Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2472,10 +2485,26 @@ - (void)takePictureOrVideo
     }];
 }
 
-- (void)chooseFromLibrary
+- (void)chooseFromLibraryAsDocument
+{
+    OWSAssertIsOnMainThread();
+
+    [self chooseFromLibrary:YES];
+}
+
+- (void)chooseFromLibraryAsMedia
+{
+    OWSAssertIsOnMainThread();
+
+    [self chooseFromLibrary:NO];
+}
+
+- (void)chooseFromLibrary:(BOOL)shouldTreatAsDocument
```

Comment:
style nit: I almost always prefer to follow apple's convention and include the parameter name in the method selector, otherwise it's not clear at the call site what the boolean means.

e.g.
```
chooseFromLibraryWithShouldTreatAsDocument:YES
```
or
```
chooseFromLibraryAsDocument:YES
```

Reply:
Done.